### ISSUE TITLE
Perhaps you may want to remove your username and password. 

### ISSUE THREAD

**Issue by 2016rshah** [Lost--/MailSatan#2]
If you create a file called `secrets.js` file and store your username and password into a variable then you can use that to authenticate with sendgrid. In order to make sure they don't get pushed to github just create a file called `.gitignore` and add the name of your secrets file. In this case that would just be `secrets.js`

**Comment by Lost--** [comment]
Yea that is a really good idea! Could you help me out with doing that? I
don't know exactly how... If you could walk me through the steps on
changing it myself, that'd be ideal.

**Comment by 2016rshah** [comment]
I will outline the steps below. I could also send you a pull request with most of it already done which I can do pretty easily if you ask me to. But it might be better if you try it out because it really isn't hard so here is how. Let me know if you have any questions or if anything is unclear. 
- Create a file called `secrets.js` 
  - `$ touch secrets.js` (or however you are used to creating files)
- Open up `secrets.js` and add the following:
  - `USERNAME = "your username goes here"`
  - `PASSWORD = "your password goes here"`
- Create a file called `.gitignore`
  - `$ touch .gitignore`
- Open up `.gitignore` and add the following:
  - `secrets.js`
- Go into where your username/password used to be (`index.js`) and:
  - Replace `var sendgrid  = require('sendgrid')("something", "something");` with `var sendgrid  = require('sendgrid')(USERNAME, PASSWORD);`

Basically all you are doing is storing them in javascript variables (that all other javascript files in the same folder can see too) and using those variables. But rather than putting the file with the sensitive info on github you are telling github to ignore it. Like I said let me know if I need to explain anything better. 

Oh and I forgot to mention: great idea haha very clever. The name is great too.

**Comment by Lost--** [comment]
So, when I made all the changes and tried to run it, it said USERNAME and
PASSWORD are not defined. What do I do to fix that?

**Comment by 2016rshah** [comment]
Okay sorry about the instructions they are wrong in this case (but will work in certain situations!). I sent in a pull request (#3) which should work but I don't have time to explain what I did differently right now. You can accept the pull request, pull the changes, and look through what changed. With the help of google things should make sense. Otherwise let me know and I will try to walk you through it step by step when I get time
